bot/exts/info/code_snippets.py

```diff
@@ -43,16 +44,13 @@ class CodeSnippets(Cog):
     Matches each message against a regex and prints the contents of all matched snippets.
     """
 
-    async def _fetch_response(self, url: str, response_format: str, **kwargs) -> str:
+    async def _fetch_response(self, url: str, response_format: str, **kwargs) -> Any:
```

Comment:
Would a return type of `Union[str, dict]` make more sense here?

Reply:
No. Technically JSON is not limited to objects (or arrays) as the root element. Plus it's an internal helper method, so I don't feel being extremely accurate is necessary, especially considering the caller literally has to pass "json" or "text" to the function.